.gitignore

```diff
@@ -8,3 +8,7 @@ custom-goose
 go.db
 goose
 sql.db
+
+
+# don't ignore goose dir
+!goose/
```

Comment:
what's in goose dir?

Reply:
the driver files

Reply:
that's `cmd/goose`, not `goose` :)

Reply:
`cmd/goose` got hit by `goose`. For some reason the driver files wouldn't show up until i removed `goose` from `.gitignore`

Reply:
remove the above "goose" from .gitignore instead? I don't know why we have that line

we should have `bin/` if we ever build local binaries